common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -343,7 +343,7 @@ else if (isRedirectUrl(formattedURL)) {
                 processWebCpEnrollmentUrl(view, url);
             } else if (mIsWebCpInWebViewFeatureEnabled && isWebCpAuthorizeUrl(url)) {
                 processWebCpAuthorize(view, url);
-            } else if (isDeviceCaRequest(url) && isHttpsScheme(url)) {
+            } else if (isDeviceCaRequest(url) && isHttpsScheme(url) && isWebCpInWebviewFeatureEnabled(url)) {
```

Comment:
>isDeviceCaRequest(url) && isHttpsScheme(url) [](http://example.com/codeflow?start=23&length=44)

Does this not apply to pre-WebCP flows (https + device ca)?

Reply:
As discussed offline, it does apply to pre-webcp flows. But I only found that the users get stuck with webcp flow enabled. In pre-webcp, they are good! So, I did not want to touch this scenario for existing customers just yet (at-least not as part of a hotfix).